ui/DesignSystem/Component/Notification.svelte

```diff
@@ -10,58 +10,55 @@
 <style>
   .notification {
     display: flex;
-    flex: 1;
-    border-radius: 2px;
-    height: 40px;
+    border-radius: 4px;
+    height: 32px;
     align-items: center;
   }
 
   .info {
-    border: 1px solid var(--color-positive);
-    color: var(--color-positive);
-    background-color: var(--color-positive-level-1);
+    color: var(--color-background);
+    background-color: var(--color-foreground);
   }
 
   .notification.info :global(svg) {
-    fill: var(--color-positive);
-  }
-
-  .warning {
-    border: 1px solid var(--color-caution);
-    color: var(--color-caution);
-    background-color: var(--color-caution-level-1);
-  }
-
-  .notification.warning :global(svg) {
-    fill: var(--color-caution);
+    fill: var(--color-background);
   }
 
   .error {
-    border: 1px solid var(--color-negative-level-6);
-    color: var(--color-negative-level-6);
-    background-color: var(--color-negative-level-1);
+    color: var(--color-background);
+    background-color: var(--color-negative);
   }
 
   .notification.error :global(svg) {
-    fill: var(--color-negative-level-6);
+    fill: var(--color-background);
   }
 
   .close {
     cursor: pointer;
-    margin: 0 8px 0 auto;
-    height: 24px;
+    margin: 0 8px 0 6px;
+    padding: 5px 0 0 8px;
+    height: 32px;
     user-select: none;
+    border-left: 1px solid;
+  }
+
+  .close.error {
+    border-color: var(--color-negative-level-2);
+  }
+
+  .close.info {
+    border-color: var(--color-foreground-level-6);
   }
 </style>
 
 <div class={`notification ${level}`} {style}>
   <svelte:component this={icon} style="margin: 8px; height: 24px" />
```

Comment:
@MeBrei How can I change the icon depending on if it's an error or a notification?

Reply:
Currently the icon is passed down by `export let icon = Icon.Info;` in `Notification.svelte`, but I think it is actually not used.
You could remove it and set the icon depending on the `level` input. Something like:
```js
const icon = level === "info" ? Icon.Info : Icon.Important;
```

Reply:
Fixed this myself